spec/features/review/user_can_see_review_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+feature "user can see reviews" do
+  scenario "user visits the release page and sees reviews" do
+    danger = User.create!(username: 'jbone91', email: 'johnmauran1@gmail.com', password: 'noneofyourbusiness')
```

Comment:
Please edit Hound for line length. See the documentation I shared in staff announcements